tests/common/port_toggle.py

```diff
@@ -110,3 +112,35 @@ def __get_down_ports():
 
     logger.info('wait %d seconds for system to startup', wait_after_ports_up)
     time.sleep(wait_after_ports_up)
+
+
+def get_port_toggle_wait_time(duthost, port_count):
```

Comment:
@Junchao-Mellanox A couple of comments here:  
No need to pass the `port_count`, you can get it from `duthost`. Also this method needs to be either refactored or new fixture provided: we can't put here all this Mellanox specific logic since it should be a common stuff. The idea is simple: an object or fixture which returns the wait timeout. Each vendor can have a dedicated object with all the relevant logic encapsulated there

Reply:
@nazarii-gnydyn As ports can be passed as an arguments, we cannot get it from `duthost`. So it is also not easy to implement this as a fixture. What do you think?

Reply:
Since the port_toggle is a function here, it is not possible to directly use a fixture here. To be able to use a fixture, it must be a test case function/method or a fixture itself.